indexer/feature_data.cpp

```diff
@@ -97,7 +98,44 @@ class UselessTypesChecker
   }
 };
 
+} // namespace
+
+namespace feature
+{
+uint8_t CalculateHeader(uint32_t const typesCount, uint8_t const geomType,
```

Comment:
And if you need you can make slight function interface modification to use it for different purposes.